packages/@o3r/application/schematics/ng-add/index.ts

```diff
@@ -67,20 +70,14 @@ function ngAddFn(options: NgAddSchematicsSchema): Rule {
         recorder,
         moduleFilePath
       );
+
       addImportToModuleFile(
         'BrowserAnimationsModule',
         '@angular/platform-browser/animations',
-        sourceFile,
-        sourceFileContent,
-        context,
-        recorder,
-        moduleFilePath,
-        moduleIndex,
-        '.withConfig({disableAnimations: prefersReducedMotion()})',
-        true
+        '.withConfig({disableAnimations: prefersReducedMotion()})'
```

Comment:
Could we add a comment above to explain that it's for accessibility ?

Reply:
as discussed, not really useful in the generated code.